sources/packages/web/src/assets/css/formio-shared.scss

```diff
@@ -341,6 +344,7 @@
   font-weight: 700 !important;
   font-size: 12px !important;
   color: $brand-black;
+  font-family: "BCSans", sans-serif;
```

Comment:
Font family is defined at the top level of HTML as shown below. Is it an accurate solution?
```
html * {
  font-family: "BCSans", "Noto Sans", Verdana, Arial, sans-serif, "primeicons";
  (...and more)
}
```

Reply:
I removed the font-family assignment here are it works perfectly 👍